media/css/mozorg/home/netneutrality-12-2017.scss

```diff
@@ -38,6 +38,7 @@
     background: #000;
     color: #fff;
     padding: 40px 20px;
+    min-height: 100vh;
```

Comment:
I had no idea viewport units we're so well supported: https://caniuse.com/#search=vh 👍